src/components/codeKeywords.tsx

```diff
@@ -0,0 +1,544 @@
+  if (typeof document === 'undefined') {
+    return null;
+  }
+
+  let portal = document.getElementById('selector-portal');
+  if (!portal) {
+    portal = document.createElement('div');
+    portal.setAttribute('id', 'selector-portal');
+    document.body.appendChild(portal);
+  }
+  return portal;
+});
+
+type KeywordSelectorProps = {
+  group: string;
+  index: number;
+  keyword: string;
+};
+
+type TokenState =
+  | {status: 'none'}
+  | {status: 'loading'}
+  | {status: 'success'; token: string}
+  | {status: 'error'};
+
+const dropdownPopperOptions = {
+  placement: 'bottom' as const,
+  modifiers: [
+    {
+      name: 'offset',
+      options: {offset: [0, 10]},
+    },
+    {name: 'arrow'},
+  ],
+};
+
+function OrgAuthTokenCreator() {
+  const {codeKeywords, sharedKeywordSelection} = useContext(CodeContext);
+
+  const [sharedSelection, setSharedSelection] = sharedKeywordSelection;
+  const [tokenState, setTokenState] = useState<TokenState>({status: 'none'});
+  const [isOpen, setIsOpen] = useState(false);
+  const [referenceEl, setReferenceEl] = useState<HTMLSpanElement>(null);
+  const [dropdownEl, setDropdownEl] = useState<HTMLElement>(null);
+  const {styles, state, attributes} = usePopper(
+    referenceEl,
+    dropdownEl,
+    dropdownPopperOptions
+  );
+
+  useOnClickOutside({
+    ref: {current: referenceEl},
+    enabled: isOpen,
+    handler: () => setIsOpen(false),
+  });
+
+  const updateSelectedOrg = (orgSlug: string) => {
+    const choices = codeKeywords.PROJECT ?? [];
+    const currentSelectionIdx = sharedSelection.PROJECT ?? 0;
+    const currentSelection = choices[currentSelectionIdx];
+
+    // Already selected correct org, nothing to do
+    if (currentSelection && currentSelection.ORG_SLUG === orgSlug) {
+      return;
+    }
+
+    // Else, select first project of the selected org
+    const newSelectionIdx = choices.findIndex(choice => choice.ORG_SLUG === orgSlug);
+    if (newSelectionIdx > -1) {
+      const newSharedSelection = {...sharedSelection};
+      newSharedSelection.PROJECT = newSelectionIdx;
+      setSharedSelection(newSharedSelection);
+    }
+  };
+
+  const createToken = async (orgSlug: string) => {
+    setTokenState({status: 'loading'});
+    const token = await createOrgAuthToken({
+      orgSlug,
+      name: `Generated by Docs on ${new Date().toISOString().slice(0, 10)}`,
+    });
+
+    if (token) {
+      setTokenState({
+        status: 'success',
+        token,
+      });
+
+      updateSelectedOrg(orgSlug);
+    } else {
+      setTokenState({
+        status: 'error',
+      });
+    }
+  };
+
+  const orgSet = new Set<string>();
+  codeKeywords?.PROJECT?.forEach(projectKeyword => {
+    orgSet.add(projectKeyword.ORG_SLUG);
+  });
+  const orgSlugs = [...orgSet];
+
+  const [isAnimating, setIsAnimating] = useState(false);
+
+  if (!codeKeywords.USER) {
+    // User is not logged in - show dummy token
+    return <Fragment>sntrys_YOUR_TOKEN_HERE</Fragment>;
```

Comment:
unrelated to your change, but do you know why this is "sntrys_YOUR_TOKEN_HERE"? 
the "sntrys" parts specifically just looks like a typo

Reply:
this is correct, the org tokens _always_ have the `sntrys_` prefix, so you can identify immediately that this is a sentry token :)